src/store/wizardSlice.ts

```diff
@@ -53,21 +53,21 @@ type wizardState = {
 
 const initialState: wizardState = {
   env: {
-    serverUrl: undefined,
-    baseUrl: undefined,
+    serverUrl: '',
+    baseUrl: '',
```

Comment:
This value is changed in the background based on the environment and it's not influenced by any possible user input. I think we can probably leave it as undefined?